_posts/2015-12-15-Automating-Testflight-Deploys.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: 'Automating TestFlight Deploys using Fastlane'
+date: 2015-12-15T00:00:00.000Z
+comments: false
+categories: [ios, mobile, devops, ci]
+author: orta
+---
+
+I've been a really [strong supporter](http://artsy.github.io/blog/2015/09/18/Cocoa-Architecture-Dependencies/) of the [Fastlane](https://fastlane.tools) toolset. I think it fixes a lot of common developer problems, in a space that Apple doesn't really touch. The command line.
+
+We've added hints of Fastlane to our apps at different rates, [Eidolon](https://github.com/artsy/eidolon/) uses Fastlane for everything but [Eigen](https://github.com/artsy/eigen/)/[Energy](https://github.com/artsy/energy)/[Emergence](https://github.com/artsy/emergence) have been pretty slow on the uptake.
```

Comment:
Could mention that Eidolon's needs are a lot smaller – no app store submission or screenshot generation, etc etc